include/bluetooth/bluetooth.h

```diff
@@ -558,6 +585,34 @@ struct bt_le_adv_param {
 					     BT_GAP_ADV_FAST_INT_MIN_2, \
 					     BT_GAP_ADV_FAST_INT_MAX_2, NULL)
 
+/** Helper to declare periodic advertising parameters inline
+ *
+ *  @param _int_min     Minimum periodic advertising interval
+ *  @param _int_max     Maximum periodic advertising interval
+ *  @param _options     Periodic advertising properties bitfield.
+ */
+#define BT_LE_PER_ADV_PARAM_INIT(_int_min, _int_max, _options) \
+{ \
+	.interval_min = (_int_min), \
+	.interval_max = (_int_max), \
+	.options = (_options), \
+}
+
+/** Helper to declare periodic advertising parameters inline
+ *
+ *  @param _int_min     Minimum periodic advertising interval
+ *  @param _int_max     Maximum periodic advertising interval
+ *  @param _options     Periodic advertising properties bitfield.
+ */
+#define BT_LE_PER_ADV_PARAM(_int_min, _int_max, _options) \
+	((struct bt_le_per_adv_param[]) { \
+		BT_LE_PER_ADV_PARAM_INIT(_int_min, _int_max, _options)) \
```

Comment:
Please check using checkpatch.... I get complaints that the parameters should be enclosed in paranthesis.

Reply:
Fixed all the checkpath warnings. I'm surprised it's not appearing here, but thanks for catching them!